packages/web-components/src/components/va-alert/va-alert.tsx

```diff
@@ -204,6 +203,34 @@ export class VaAlert {
             <va-icon icon="cancel" size={4}></va-icon>
           </button>
         )}
+      </Fragment>
+    );
+
+    if (fullWidth) {
+      return (
+        <Host>
+          <section
+            class={classesSiteAlert}
+            aria-label={this.el.getAttribute('data-label')}
+            role={this.el.getAttribute('data-role')}
```

Comment:
I'm making an assumption that we still want `aria-label` and `role` on the parent element but I'm not totally sure that's correct now because the full width version is wrapped in a `section` element instead of a `div`.